jobs/JGDAS_GLOBAL_MARINE_ANALYSIS_LETKF

```diff
@@ -12,9 +12,14 @@ GDATE=$(date --utc +%Y%m%d%H -d "${PDY} ${cyc} - ${assim_freq} hours")
 gPDY=${GDATE:0:8}
 gcyc=${GDATE:8:2}
 
+export GDUMP=${GDUMP:-"gdas"}
+export GDUMP_ENS="enkf${GDUMP}"
+
 YMD=${gPDY} HH=${gcyc} declare_from_tmpl -rx \
```

Comment:
When this is run in the early cycle, `RUN` will be `enkfgfs`, but the backgrounds will be from `enkfgdas`.
I think we will need a `RUN=enkfgdas` regardless of what the cycle (`RUN`) this is run in.

Reply:
What should I be using to define the observation filename? It's currently 
`obs_filename = f"{self.task_config.RUN}.t{self.task_config.cyc}z.{obs_name}.{to_YMDH(self.task_config.current_cycle)}.nc4"`